proxy/package.json

```diff
@@ -14,14 +15,18 @@
     "serve-legacy": "cd ../legacy && yarn run start",
     "serve-root": "cd ../root && yarn start",
     "serve-ui": "cd ../ui && BROWSER=none PORT=8401 yarn run start",
+    "serve-ui-proxy": "node ./ui.js",
+    "serve-ui-standalone": "yarn build-root-css && cd ../ui && BROWSER=none PORT=8401 yarn run standalone",
     "start": "concurrently \"yarn serve-apps\" \"yarn serve-base\"",
+    "start-ui": "concurrently \"yarn serve-ui-standalone\" \"yarn watch-shared\" \"yarn serve-ui-proxy\"",
     "watch-shared": "cd ../shared && yarn watch"
   },
   "devDependencies": {
     "concurrently": "5.2.0",
     "cypress": "4.5.0",
     "dotenv-flow": "3.1.0",
     "http-proxy-middleware": "1.0.3",
+    "node-sass-tilde-importer": "^1.0.2",
```

Comment:
Doesn't npmPackageJsonLint usually complain about this?

Reply:
Yeah, I'm not sure why that didn't complain... I'll fix it.